web/src/js/utils/useTrueX.js

```diff
@@ -127,53 +113,47 @@ const useTrueX = ({
         })
         if (success) {
           setCredited(true)
+        } else {
+          setError(true)
         }
       })
 
       // User closed the ad unit.
       trueX.ad.onClose(activity => {
-        log('closed')
         setStatus(CLOSED)
       })
 
       // User got to end of ad.
       trueX.ad.onFinish(activity => {
-        log('finished')
         setStatus(COMPLETED)
       })
 
-      trueX.ad.onMessage(payload => {
-        log('onMessage = ' + payload)
-      })
+      trueX.ad.onMessage(payload => {})
 
       // "Triggered when an error has occurred with the ad.
       // This should be considered an exception. It's best
       // practice to remove the ad container when this occurs."
       // https://github.com/socialvibe/truex-ads-docs/blob/master/js_ad_api.md#ad-object
       trueX.ad.onError(error => {
-        log('error = ' + error)
-
-        // The parent should close the container.
-        setStatus(CLOSED)
+        // show error message, user can click away
+        setError(true)
       })
-
-      // Set that true[X] is ready to go.
-      setStatus(READY)
-      setAdAvailable(true)
+      // Set that true[X] is ready to go from waiting.
+      if (status === WAITING) {
```

Comment:
this may also have been what caused a bug.  This setStatus back to ready was being called the same time ad.onStart was firing